src/components/layout/MergeDataChallengeLayout.tsx

```diff
@@ -0,0 +1,16 @@
+import { Stack } from '@chakra-ui/react';
```

Comment:
I think it would be better to refactor all the hero components to be a single component and keep these to layout components for the different layouts to import images and text specific to that layout. What do you think?